Snippets/ASBFunctions/ASBFunctions_1.3/FakeFunctionsSdk.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+
+public interface IActionResult
+{
+}
+
+class OkObjectResult : IActionResult
+{
+    public OkObjectResult(string s) => throw new NotImplementedException();
+}
+class TriggerMessage
+{
+}
+
+class HttpTrigger : Attribute
+{
+    public HttpTrigger(AuthorizationLevel authorizationLevel, string a, string b)
+    {
+    }
+
+    public string Route { get; set; }
+}
+
+public class HttpRequest
+{
+}
+
+enum AuthorizationLevel
+{
+    Function
+}
```

Comment:
Why do this and not just reference the real sdk?

Reply:
don't know exactly...possibly because we didn't want to install other SKDs (Azure etc) on the build engine? The older versions are also using stubs like this.

Reply:
if it's possible, using the real SDK seems to make more sense unless that breaks something?

Reply:
used the SDK and removed all the stubs.